doc/examples/bundle_assignment_maps.py

```diff
@@ -80,9 +80,9 @@
 
 scene = window.Scene()
 scene.SetBackground(1, 1, 1)
-scene.add(actor.streamtube(model_af_l, colors=disks_color))
+scene.add(actor.line(model_af_l, colors=disks_color))
```

Comment:
This looks a bit nicer:
```suggestion
scene.add(actor.line(model_af_l, colors=disks_color, fake_tube=True, linewidth=6))
```